### ISSUE TITLE
2025-12-04: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#481]
# 每日更新报告（2025-12-04）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-12-04 23:38:03 | Doug Smith | Mark DBO test as flaky on b200 for Distributed B200 test (#29913) |
| 2025-12-04 23:19:42 | Woosuk Kwon | [Model Runner V2] Implement get_num_sampled_and_rejected kernel (#30029) |
| 2025-12-04 22:54:28 | Harry Mellor | Use Transformers v5 RoPE standardisation and validation (#30046) |
| 2025-12-04 22:04:42 | Yongtao Huang | Fix broken multiline assert in `LoRAModelManager.register_module` (#30032) |
| 2025-12-04 21:44:50 | Harry Mellor | Delete HF version of Phi 4 MM (#30049) |
| 2025-12-04 21:44:15 | wang.yuqi | [Model][6/N] Improve all pooling task \| Support chunked prefill with ALL pooling (#27145) |
| 2025-12-04 19:18:29 | Kevin H. Luu | [release] install regex (#30008) |
| 2025-12-04 19:11:59 | Chauncey | [Bugfix] Fix the issue with interleaved thinking when using streaming (#30033) |
| 2025-12-04 19:00:16 | Andreas Karatzas | [ROCm][CI][Bugfix] Fixing the `Multi-Modal Models Test (Extended) 1` group (#30013) |
| 2025-12-04 18:04:43 | Noa Neria | Validating Runai Model Streamer Integration with S3 Object Storage (#29320) |
| 2025-12-04 17:51:36 | dtc | [P/D] Introduce Mooncake Transfer Engine as kv_connector (#24718) |
| 2025-12-04 17:30:22 | rasmith | [CI/Build][AMD] Skip test on test_hybrid_attention_mamba_tensor_shapes on ROCm, requires FLASHINFER (#29995) |
| 2025-12-04 17:15:04 | Arpit Khandelwal | [Core] Remove forced None assignment for deprecated PassConfig flags (#29994) |
| 2025-12-04 16:46:34 | Xu Wenqing | Add DeepSeek-V3.2 tool parser. (#29848) |
| 2025-12-04 16:45:57 | Ye (Charlotte) Qi | [ROCm] add fallback for aiter fp8 decode mla (#30005) |
| 2025-12-04 16:22:03 | Mark McLoughlin | [Core] Fix standalone runs of test_reset_prefix_cache_e2e (#29899) |
| 2025-12-04 16:21:19 | Cyrus Leung | [Misc] Move functions into `PoolingMetadata` (#30027) |
| 2025-12-04 16:20:54 | Micah Williamson | [CI][AMD] Match Main CI Behavior By Skipping test_eplb_spec_decode In AMD CI (#30006) |
| 2025-12-04 16:20:24 | TJian | [ROCm] [Bugfix] `compute_attn_mask_seqlen` for qwen3 omni (#29974) |
| 2025-12-04 15:52:28 | Charlie Fu | [Rocm][CI] Fix test_speculator_eagle3 by skipping the CompressedTensorw4a16 Model (#30001) |
| 2025-12-04 15:46:15 | CYJiang | [docs] Remove _total from counter metrics names (#30028)<br>In Prometheus Counters always expose their actual numeric value with a metric name that ends in _total. We should document the base name, as this what appears in the get_metrics() API. |
| 2025-12-04 15:36:57 | daniel-salib | [Frontend] refactor harmony utils output message parsing (#29820) |
| 2025-12-04 15:12:12 | Chauncey | [Bugfix] fixed deepseekv32 tool calling error (#30025) |
| 2025-12-04 14:22:20 | Cyrus Leung | [Misc] Various cleanups for MM input processing (#29970) |
| 2025-12-04 14:04:44 | Jianwei Mao | Fix error while downloading dependencies for CPU backend (#29797) |
| 2025-12-04 14:00:52 | Kuntai Du | [KVConnector] remove unused code (the model aware kv ops class) (#29709) |
| 2025-12-04 13:50:27 | Benjamin Bartels | [Frontend] Fixes anthropic /v1/messages streaming not containing input_tokens on first chunk (#29971) |
| 2025-12-04 13:01:15 | gausah01 | [Bugfix][CPU] Fix CPU KV cache fallback memory allocation (#29604) |
| 2025-12-04 12:51:15 | Jee Jee Li | [Bugfix] Fix adapter_enabled IMA (#29977) |
| 2025-12-04 11:53:00 | Li Wang | [Misc] Add docker build env for Ascend NPU (#30015) |
| 2025-12-04 11:48:30 | Iceber Gu | fix LoRA-related examples (#29956) |
| 2025-12-04 11:21:45 | Zhewen Li | [CI/Build] Add MM code path to Examples Test (#29986) |
| 2025-12-04 09:02:02 | Xieyang Xu | enable multi-node in external launcher mode (#29833) |
| 2025-12-04 06:08:19 | Shengqi Chen | [CI] fix docker image build by specifying merge-base commit id when downloading pre-compiled wheels (#29930) |
| 2025-12-04 06:00:52 | Elizabeth Thomas | [Bugfix] Respect VLLM_CONFIGURE_LOGGING value (#28671) |
| 2025-12-04 04:49:00 | bnellnm | [Kernels] Remove BatchedTritonOrDeepGemmExperts and default fallback to Triton (#29929) |
| 2025-12-04 04:16:54 | Wentao Ye | [CI] Fix re import error (#29973) |
| 2025-12-04 04:14:44 | Yongtao Huang | Fix LLMEngine.del dp_group cleanup condition (#29954) |
| 2025-12-04 02:50:53 | elvischenv | [Bugfix] Fix flashinfer ar+norm kernel not available issue (#29960) |
| 2025-12-04 02:04:59 | Varun Sundar Rabindranath | [Performance][DP/EP] Add silu_mul_per_token_group_quant_fp8_colmajor kernel (#29470) |
| 2025-12-04 01:45:31 | avigny | [Bugfix] Mistral tool parser streaming update (#19425) |
| 2025-12-04 01:17:07 | Micah Williamson | [ROCm][CI] Fix v1/logits_processors failure on ROCm (#29927) |
| 2025-12-04 00:12:58 | Yu Jiaqi | SigLIP example add chat_template (#29902) |
| 2025-12-04 00:06:57 | Lumis Chen | [Core] Add xxHash as a high-performance hash option for accelerating prefix caching (#29163) |
